package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cumulus/core",
-  "version": "1.0.0",
+  "version": "1.0.1-0",
```

Comment:
Is there a reason for not using a -alpha designation here?

Reply:
This was unintentional. I'll revert